data/remnant/remnant 2 cognizance.txt

```diff
@@ -696,7 +696,7 @@ mission "Remnant: Cognizance 20"
 	stopover
 		government "Remnant"
 		neighbor distance 1 5
-		# Without the following line, the stopover planet will occasionally be the origin planet. I don't know why, but that's the way it is
+		# Without the following line, the stopover planet will occasionally be the origin planet. I do not know why, but that is the way it is
```

Comment:
It's not an unsolved issue
`neighbor distance 1 5`
Means that, in order for a planet or system to match this location filter, a system it is connected directly to by a hyperlink must be 1 to 5 (inclusive) hyperjumps away from the `origin` (the `source` of this mission).
This is true for the origin system and any planets within it, so it can select those locations.
The `not distance 0` excludes the origin system.

Bas location filters like this are one of the things I fix in #7261.